### PR TITLE
Adds chain id to constructor

### PR DESCRIPTION
**Pull request by kaw2k**
Context: 
We have an issue where a wallets primary name doesn't show up in the wallet connector if they are connected via IC. Doing some digging, it looks like [we don't pass a network chain id](https://github.com/aptos-labs/aptos-wallet-adapter/blob/1470bb9b3d8c51407d70aa994624d672a36751f0/packages/wallet-adapter-core/src/WalletCore.ts#L125) to this function. Looking at [how it is done in petra](https://github.com/aptos-labs/petra-plugin-wallet-adapter/blob/68fced6dcde1d5f2ab9685c21c9c2dcbbda7a21f/src/index.ts#L239), this is returned via calling the wallet's getNetwork call. [IC's version of this function doesn't have this value](https://github.com/aptos-labs/ic-plugin-wallet-adapter/blob/fa2b65a0a6a6a4bd19d408e73aaedb3d73e94767/src/index.ts#L236). This PR adds chain id as an optional argument to the constructor, and defaults to the hardcoded values based on the network provided.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -18,6 +18,11 @@ import { BCS, HexString, TxnBuilderTypes, Types } from 'aptos';
 import { txnAuthenticatorFromAccountAuthenticator } from './normalization';
 
 type ICAccount = Awaited<ReturnType<ICDappClient['getConnectedAccounts']>>[0];
+const ChainIds: Record<NetworkName, string> = {
+  mainnet: "0",
```
Comment (hardsetting) [inline src/index.ts]: I think mainnet is 1 and testnet is 2?

**Review by hardsetting — APPROVED** [review]
LGTM, just double check the actual values :P

Although tbh I think the wallet shouldn't have to return the chainId and should be optional, and the wallet adapter can fetch it automatically by calling the API

File: src/index.ts [hunk 2]
```diff
@@ -18,6 +18,11 @@ import { BCS, HexString, TxnBuilderTypes, Types } from 'aptos';
...
+  testnet: "1",
+  devnet: "", // Not supported
```
Comment (hardsetting) [inline src/index.ts]: can this be undefined instead?

File: src/index.ts (hunk 1 above)
Comment (kaw2k) [reply src/index.ts]: You are correct! My brain just went right to 0 based indexing :joy:

**Review by hardsetting — APPROVED** [review]
Awesome LGTM

**Pull request merged**